WordPress/Classes/ViewRelated/Stats/Shared Views/StatsCellHeader.swift

```diff
@@ -35,10 +36,17 @@ class StatsCellHeader: UITableViewCell, NibLoadable, Accessible {
     }
 
     func prepareForVoiceOver() {
-        isAccessibilityElement = !(headerLabel.text?.isEmpty ?? true)
-        accessibilityElementsHidden = (headerLabel.text?.isEmpty ?? true)
-        accessibilityLabel = headerLabel.text
-        accessibilityTraits = .staticText
+        headerLabel.isAccessibilityElement = !(headerLabel.text?.isEmpty ?? true)
+        headerLabel.accessibilityElementsHidden = (headerLabel.text?.isEmpty ?? true)
```

Comment:
What if you set the bool value as `headerLabel.text?.isEmpty == false` and `headerLabel.text?.isEmpty == true`?

Reply:
Done!